pkg/util/jsonrawobjectwriter/json_raw_object_writer.go

```diff
@@ -0,0 +1,121 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Instead of a pkg `util/jsonrawobjectwriter`, I think it would be great to have an util pkg simply called `util/json`: it would contain your file `json_raw_object_writer.go` and would let people manipulate from the outside the `json.JSONRawObjectWriter` struct or use the `json.NewJSONRawObjectWriter(...)` constructor. (for example, same goes with `json.OmitEmpty` which feels natural while calling `AddStringField`).

WDYT?